LICENSE.md

```diff
@@ -0,0 +1,25 @@
+web-mapviewer is available for use under the following license, commonly known
+as the 3-clause (or "modified") BSD license:
+
+==============================
+
+Copyright (c) 2019-2020, swisstopo
```

Comment:
I thought we would write 2020 only, but if we go the way of multiple years I would prefer to write `2019-present` so we don't have to go back each new year's eve to edit this year

Reply:
ok 2020 only